reporter/src/main/kotlin/gitlab/GitlabLicenseModel.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter.gitlab
+
+/**
+ * The Gitlab license model according to
+ * https://gitlab.com/gitlab-org/security-products/license-management/-/blob/master/spec/fixtures/schema/v2.1.json,
```

Comment:
Please use permalinks to document the exact state of the files used (also below, and in the commit message).

Reply:
Done.